packages/medusa/src/services/product-category.ts

```diff
@@ -116,6 +119,33 @@ class ProductCategoryService extends TransactionBaseService {
     })
   }
 
+  /**
+   * Updates a product category
+   * @param productCategoryId - id of product category to update
+   * @param productCategoryInput - parameters to update in product category
+   * @return updated product category
+   */
+  async update(
+    productCategoryId: string,
+    productCategoryInput: UpdateProductCategoryInput
+  ): Promise<ProductCategory> {
+    return await this.atomicPhase_(async (manager) => {
+      const productCategoryRepo = manager.getCustomRepository(
+        this.productCategoryRepo_
+      )
+
+      const productCategory = await this.retrieve(productCategoryId)
+
+      for (const key in productCategoryInput) {
+        if (isDefined(productCategoryInput[key])) {
+          productCategory[key] = productCategoryInput[key]
+        }
+      }
```

Comment:
suggestion(non-blocking): The repositories can merge entities as so: 
```suggestion
      productCategoryRepo.merge(productCategory, productCategoryInput)
```

Reply:
nice, didn't know this!